elementary/monitor/dbt_project/models/tests_recommendation/table_timestamp_columns.sql

```diff
@@ -1,35 +1,35 @@
+{# Prioritization: 1. insertion time, 2. update time. #}
 {% set timestamp_column_names = [
+    "timestamp",
```

Comment:
I think this is also likely an event / data timestamp

Reply:
Deleted.